### PR TITLE
[914] Update README

### PR DESCRIPTION
**Pull request by darokel**
### Context

Our README needs a little bit of re-organisation given the repo has gone through a lot of changes.

### Changes proposed in this pull request

- Update README to reflect new architecture and decisions
- Split things out so it's easier to track information down

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,250 +1,59 @@
...
-### Trust the TLS certificate
+# Find & Publish Teacher Training
 
-Depending on your browser you may need to add the automatically generated SSL
-certificate to your OS keychain to make the browser trust the local site.
+This repo is home to three services:
 
-On macOS:
+- A service for candidates to [find teacher training](https://www.find-postgraduate-teacher-training.service.gov.uk)
+- A service for providers to [publish teacher courses](https://www.find-postgraduate-teacher-training.service.gov.uk)
```
Comment (avinhurry) [inline README.md:6]: copy paste...

File: README.md [hunk 2]
```diff
@@ -1,250 +1,59 @@
...
 
-```bash
-travis encrypt AZURE_CR_PASSWORD="xxx" --add
-```
 
-## Sentry
+## Guides
 
-To track exceptions through Sentry, configure the `SENTRY_DSN` environment variable:
+- [Configuration](/docs/configuration.md)
```
Comment (avinhurry) [inline README.md:41]: What do you think about organising these files into a folder called "guides"?

Comment (avinhurry) [reply README.md:41]: I wish there was a way we could just click on those links 🙃

File: docs/authentication.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Authentication
+
+## Find
```
Comment (avinhurry) [inline docs/authentication.md:3]: Worth adding a line here to say that Find is publically available, but certain parts of it are behind basic auth?

File: docs/setup-development.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
+# Installation
+
+Clone the repo:
+
+    git clone git@github.com:darokel/bento.git
```
Comment (avinhurry) [inline docs/setup-development.md:5]: 🙈

File: docs/setup-development.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+## Setup the application
+
+Run the following commands:
+
+```bash
+yarn
+bundle
+bundle exec rails db:setup
+```
```
Comment (avinhurry) [inline docs/setup-development.md:16]: I think it'd be useful to have something about populating the local database correctly too

**Review by avinhurry — COMMENTED** [review]
Looking good!

File: docs/setup-development.md (hunk 4 above)
Comment (darokel) [reply docs/setup-development.md:5]: 🤣

**Comment by darokel** [comment]
[Update the repo’s README to reflect the new architecture](https://trello.com/c/c5G9PY3c/914-update-the-repos-readme-to-reflect-the-new-architecture)

File: docs/setup-development.md (hunk 5 above)
Comment (darokel) [reply docs/setup-development.md]: db:setup runs `seed` with some test data but have included some steps to seed a sanitised dump

File: docs/authentication.md (hunk 3 above)
Comment (darokel) [reply docs/authentication.md]: done

File: README.md (hunk 2 above)
Comment (darokel) [reply README.md]: Good idea. You should be able to on Github and locally in your editor? (if it supports it)

File: README.md (hunk 1 above)
Comment (darokel) [reply README.md]: fixed

File: README.md [hunk 6]
```diff
@@ -1,250 +1,59 @@
...
-### Trust the TLS certificate
+# Find & Publish Teacher Training
 
-Depending on your browser you may need to add the automatically generated SSL
-certificate to your OS keychain to make the browser trust the local site.
+This repo is home to three services:
 
-On macOS:
+- A service for candidates to [find teacher training](https://www.find-postgraduate-teacher-training.service.gov.uk)
+- A service for providers to [publish teacher courses](https://www.publish-teacher-training-courses.service.gov.uk)
```
Comment (avinhurry) [inline README.md:6]:
```suggestion
- A service for providers to [publish teacher training courses](https://www.publish-teacher-training-courses.service.gov.uk)
```

File: guides/authentication.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# Authentication
+
+## Find
+
+The Find servie is publicly available with one or two internal routes protected by basic auth.
```
Comment (avinhurry) [inline guides/authentication.md:5]: typo

**Review by avinhurry — APPROVED** [review]
A couple of typos, but otherwise looks good to me

**Pull request merged**